utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -99,6 +103,7 @@ object SwaggerTyped {
       Typed.typedClass[LocalDate]
     case SwaggerTime =>
       Typed.typedClass[LocalTime]
+    case SwaggerUnion(types) => Typed(types.map(typingResult).toSet)
```

Comment:
how will the final type in completion look like ?

Reply:
`Integer | String` - the same as in avro